src/store/actions/types.js

```diff
@@ -2,15 +2,22 @@ const FETCH_ORGANIZATION = 'ORGANIZATION_FETCHED'
 const FETCH_ORGANIZATION_PENDING = 'ORGANIZATION_FETCHED_PENDING'
 const FETCH_ORGANIZATION_FULFILLED = 'ORGANIZATION_FETCHED_FULFILLED'
 const FETCH_ORGANIZATION_REJECTED = 'ORGANIZATION_FETCHED_REJECTED'
+
 const FETCH_EVENTS = 'EVENTS_FETCHED'
 const FETCH_EVENTS_PENDING = 'EVENTS_FETCHED_PENDING'
 const FETCH_EVENTS_FULFILLED = 'EVENTS_FETCHED_FULFILLED'
 const FETCH_EVENTS_REJECTED = 'EVENTS_FETCHED_REJECTED'
+
 const FETCH_EVENT_DETAILS = 'EVENT_DETAILS_FETCHED'
-const FETCH_EVENT_DETAILS_PENDING = 'EVENT_FETCHED_PENDING'
+const FETCH_EVENT_DETAILS_PENDING = 'EVENT_DETAILS_FETCHED_PENDING'
 const FETCH_EVENT_DETAILS_FULFILLED = 'EVENT_DETAILS_FETCHED_FULFILLED'
 const FETCH_EVENT_DETAILS_REJECTED = 'EVENT_DETAILS_FETCHED_REJECTED'
 
+const FETCH_PAST_EVENTS = 'PAST_EVENTS_FETCHED'
```

Comment:
Shouldn't this be `const FETCH_PAST_EVENTS = 'FETCH_PAST_EVENTS'`?

I know all the above types follow this approach, but I've changed it at #30

Reply:
👍